### ISSUE TITLE
Does `operator-sdk build` support ppc64le

### ISSUE THREAD

**Issue by vincent-pli** [operator-framework/operator-sdk#1595]
I mean could `operator-sdk build` build go binary for `ppc64le`, local build or cross-compile.
BTW, what about `s390x`

**Comment by vincent-pli** [comment]
Hi folks, I checked the code in `cmd/operator-sdk/build/cmd.go`, see this:
```
	goBuildEnv := append(os.Environ(), "GOOS=linux", "GOARCH=amd64")
```
Could we remove the `"GOARCH=amd64"`?

**Comment by joelanford** [comment]
I think we should support other architectures, but I think we should be careful how we do it. As the code currently stands, it will always automatically cross-compile to `amd64` even when run from non-amd64 machines.

Probably the best way to handle this would be to check if `GOARCH` is explicitly set. If it is, use it, and if it isn't keep the default of `amd64`.

Would that work for you? If so, you would be interested in submitting a PR with that patch?

**Comment by vincent-pli** [comment]
@joelanford  
Sure, I'm glad to help on this issue

**Comment by joelanford** [comment]
Thanks @vincent-pli 

Closed by #1605

**Comment by chenzhiwei** [comment]
I am using this place to store operator-sdk ppc64le binary.

[operator-sdk.zip](https://github.com/operator-framework/operator-sdk/files/3650671/operator-sdk.zip)
